extensions/amp-story/1.0/test/test-amp-story.js

```diff
@@ -1816,5 +1816,41 @@ describes.realWin(
         toggleExperiment(win, 'amp-story-branching', false);
       });
     });
+
+    describe('amp-story play/pause', () => {
+      it('should set playable to true if page has autoadvance', async () => {
+        await createStoryWithPages(1, ['cover'], true);
```

Comment:
nit: can you please add `true /** autoAdvance */` in the boolean arguments to follow the style guide